basiclibrary/src/main/java/Library.java

```diff
@@ -79,10 +80,45 @@ public static void calculateAverages(int[][] monthTempsByWeek) {
         return highLow;
     }
 
-    public static String tally() {
+    public static String tally(ArrayList votes) {
+        HashMap<String, Integer> voteTally = new HashMap<>();
+        for (String vote : votes ) {
+            if (voteTally.containsKey(vote)) {
+                voteTally.put(vote, voteTally.get(vote) + 1);
+            }
+            else {
+                voteTally.put(vote, 1);
+            }
+        }
 
+        int voteTotal = 0;
+        String winner = "";
+        for (String key : voteTally.keySet()) {
+            if (voteTally.get(key) > voteTotal) {
+                voteTotal = voteTally.get(key);
+                winner = key;
+            }
+        }
 
+        System.out.println(votes);
+        System.out.println("The " + winner + " received the most votes!");
+        return winner;
     }
 
+    public static void linter(Path readerFile) {
+        try(BufferedReader reader = Files.newBufferedReader(readerFile)) {
```

Comment:
Double check how you're calling this method. Is it a property of File or Files? look for errors as you type in the editor.